website/client/src/pages/private-messages.vue

```diff
@@ -630,8 +648,13 @@ export default {
 
     await this.reload();
 
-    const data = this.$store.state.privateMessageOptions;
+    // close members modal if the Private Messages page is opened in an existing tab
+    if (this.hasPrivateMessageOptionsOnPageLoad) {
+      this.$root.$emit('habitica::dismiss-modal', 'members-modal'); // hide members modal on linux
```

Comment:
this would be needed once you also update the `membersModal.vue` :P 

but for now `profile` would be the better solution

Reply:
@negue are you suggesting to remove the line above or to simply change it to `this.$root.$emit('habitica::dismiss-modal', 'profile');`?

I'm also wondering if the line below is actually needed